Kozariz.Edelveis.EF.Database/Services/UserService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Kozariz.Edelveis.Models.UsersTable;
+using Kozariz.Edelveis.EF.Database;
+
+namespace Kozariz.Edelveis.Core.Services
+{
+    public class UserService : IUserService
```

Comment:
The UserService class could benefit from additional error handling and validation to handle potential exceptions and ensure data integrity.

Reply:
Right, I will handle this